specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/examples/NameSpaces/VirtualNetworkRule/SBNetworkRuleSetCreate.json

```diff
@@ -0,0 +1,103 @@
+{
+  "parameters": {
+    "namespaceName": "sdk-Namespace-6019",
+    "resourceGroupName": "ResourceGroup",
+    "api-version": "2017-04-01",
+    "subscriptionId": "Subscription",
+    "parameters": {
+      "properties": {
+        "defaultAction": "Deny",
+        "virtualNetworkRules": [
```

Comment:
>virtualNetworkRules [](start = 9, length = 19)

This is taking arrays.. PUT request MUST replace server-side representation of the resource.. should NOT merge.

Reply:
you can consider PATCH support if support for partial updates (additions) are desired. 

---
In reply to: [265808072](https://github.com/Azure/azure-rest-api-specs/pull/5361#discussion_r265808072) [](ancestors = 265808072)

Reply:
yes, its replaces the resource. we are considering the PATCH support in future releases.